src/pages/docs/releases/channels/index.md

```diff
@@ -83,6 +89,53 @@ The **Design Version Rule** window will show a list of the packages that will de
 
 6. Click **SAVE**.
 
+### Git resource rules
+
+:::div{.hint}
+Support for Git resource rules is currently rolling out to Octopus Cloud.
+:::
+
+Git resource rules assist in ensuring that only the correct branches and tags can be used in the channel for steps that source files from an external Git repository. They are used when creating a release, either manually or via [project triggers](/docs/projects/project-triggers).
+
+1. From the **New Channel** screen, click **Add Rule** in the Git Resource Rules section.
+2. Select the step(s) that use external Git repositories the rule will be applied to.
+3. Enter patterns (separated by commas) to restrict which branches and/or tags can be selected when creating releases. Wildcard characters can be used, see [Glob patterns in Git rules](#git-rules-glob-patterns) for more information.
+4. Click **Save**.
+
+:::figure
+![Git resource rules example](/docs/releases/channels/images/git-resource-rules.png)
+:::
+
+### Git reference rules (version controlled projects)
+
+:::div{.hint}
+Support for Git reference rules is currently rolling out to Octopus Cloud.
+:::
+
+Git reference rules assist in ensuring that only the correct branches and tags can be used as the Git reference for a release in the channel for version controlled projects. They are used when creating a release, either manually or via [project triggers](/docs/projects/project-triggers).
```

Comment:
`the correct branches and tags`

Do you think configured would be more fitting here than correct?

Reply:
Yep that does sound better thanks